lib/client/configurator/configurator.go

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
`configurator` is a bit too generic, perhaps `plugins` is a better name? or `extensions`?